src/scss/_breakpoints.scss

```diff
@@ -0,0 +1,36 @@
+@import "settings";
+
```

Comment:
Make the media queries more readable by following the [Landon Schropp’s technique](https://css-tricks.com/approaches-media-queries-sass/#aa-landon-schropps-technique) (discovered this by chance :smile:)

Reply:
Maybe we should suggest to this to Vanilla.